app/config/config.php

```diff
@@ -15,36 +15,33 @@
     $db['scheme'] = 'mysql';
 }
 
-$projectDir = dirname(dirname(__DIR__)) . DIRECTORY_SEPARATOR;
-
-return array_replace_recursive($this->loadConfig($this->AppPath() . 'Configs/Default.php'), [
-
-    'db' => [
-        'username' => $db['user'],
-        'password' => $db['pass'],
-        'dbname'   => $db['path'],
-        'host'     => $db['host'],
-    ],
+if ('dev' === getenv('SHOPWARE_ENV')) {
```

Comment:
The composer project should not dictate any environment specific behavior.
The composer project is a template for own shopware based projects which can define environments as they please.

Reply:
That might be correct but I'd like to define the behaviour in the .env file so I could deploy the same files on different environments.

Maybe the 'dev'-flag is too simple for many configurations but I think there should be a possibility to define the whole config (or the relevant environment parts of it) in an .env-file.

If I'm not getting the point please tell me.